freqtrade/constants.py

```diff
@@ -89,7 +89,30 @@
             'properties': {
                 'use_sell_signal': {'type': 'boolean'},
                 'sell_profit_only': {'type': 'boolean'},
-                "ignore_roi_if_buy_signal_true": {'type': 'boolean'}
+                'ignore_roi_if_buy_signal_true': {'type': 'boolean'},
+                'check_depth_of_market': {
+                    'type': 'object',
+                    'properties': {
+                        'enabled': {'type': 'boolean'},
+                        'bids_to_ask_delta': {'type': 'number', 'minimum': 0},
+                    }
+                },
+                'bid_strategy': {
```

Comment:
I think we should be carefull with naming of this property - there is a field `"bid_strategy"` in config already (not in `"experimental"`) - and i had to think (fail) twice which one is the correct to change.

The 2 are directly related (actually do the same thing but differently), maybe it would be a good idea to move this from experimental to "bidstrategy"? 
(I don't like the experimental section anyway - as most not "really" experimental anymore - but changing that shall not be subject of this PR).

Reply:
If its ok to move it to bid_strategy, then it should be fine. I can create a commit to move it out of experimental while this PR is not yet merged. What do you think?

Reply:
i would appreciate that, yes - it is confusing to have the same setting twice.